### ISSUE TITLE
0.0.8-05 downloads silently fail

### ISSUE THREAD

**Issue by ilovezfs** [Evolution0/bandcamp-dl#120]
**Python version:**  
Python 3.6.1
**Bandcamp-dl version:**
0.0.8-05
**Bancamp-dl options:**  
`bandcamp-dl --artist=iamsleepless --album=rivulets`

**Describe the issue:**
-------------------------
The download does not happen but a not.finished file is created
```
bash-3.2$ cat 0.0.8-05.not.finished 
{'--album': 'rivulets', '--artist': 'iamsleepless', '--base-dir': None, '--embed-art': False, '--embed-lyrics': False, '--full-album': False, '--group': False, '--help': False, '--no-art': False, '--no-slugify': False, '--overwrite': False, '--template': '%{artist}/%{album}/%{track} - %{title}', '--track': None, '--version': False, 'URL': None}bash-3.2$ 
```

**Comment by Evolution0** [comment]
Indeed it does.. I really need to get verbose mode in.

Looking into it.

**Comment by ilovezfs** [comment]
@Evolution0 Thanks!

**Comment by Evolution0** [comment]
Figured it out, had to do with how I handle checking that the user actually used valid arguments, I was only checking if a url was provided (which would not be the case with `--artist`, `--track`, `--album`)

This was also why the not.finished file did not include a url even though it was generated properly.

In the process I just went ahead and put the `--debug` option in, that way figuring these kinds of things out can be done faster in the future.

New release pushed to PyPI and the releases section. Update and all should be good.
